src/gridiron/base_classes/page.cpp

```diff
@@ -345,7 +344,7 @@ Page::render() {
         lastpos = endpos + 2;
     }
 
-    return rendered;
+    return oatpp::String(rendered.c_str());
```

Comment:
At the moment it seems to be the only dependency on oatpp in the gridiron.
I think it might be a good idea to use `std::string` here and make the conversion on the endpoint side.

Thus gridiron will be more portable across apps and different web-frameworks.

Reply:
That's an excellent point- though I think I've found my home.
That being said, I may well split the gridiron example code out from the library (which I'd leave behind in this repo). The CMake structure is already mostly ready for it.